lib/dispatcher/get-model.js

```diff
@@ -20,9 +20,5 @@ module.exports = async ({ schema, licenceType, task }) => {
       break;
   }
 
-  if (!model) {
-    throw new Error(`unable to fetch data for ${licenceType} - id: ${modelId}`);
```

Comment:
What's the reason for removing this?

Reply:
because it was throwing errors on things that didn't have all the things defined (new models etc) - we don't want these things 100% of the time so shouldn't throw errors if they aren't there

Reply:
ah you removed the whitelist - https://github.com/UKHomeOffice/asl-notifications/commit/985f0568c7703d2c392d660d62f5129d25e5163c

IMO that was useful because it served as a reminder to either ignore the new model or add notifications for it.